Composer/packages/client/src/components/Modal/dialogStyle.ts

```diff
@@ -1,10 +1,30 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+import { css } from '@emotion/core';
+
 /**
  * Well known style names for the AlertDialog and ConfirmDialog.
  */
 export const dialogStyle = {
   normal: 'NORMAL',
   console: 'CONSOLE',
 };
+
+export const NORMAL_STYLE = css`
```

Comment:
Is this naming casing a standard convention for styles?

Reply:
I use CAPS_WITH_UNDERSCORES as a convention for constants. Do we have a team standard for these?